tensorly/contrib/sparse/backend/tensorflow_backend.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+import tensorflow as tf
+from ....backend.core import Backend
+from copy import copy as _py_copy
+
+
+def is_sparse(x):
+    return isinstance(x, tf.sparse.SparseTensor)
+
+
+class TensorflowSparseBackend(Backend):
+    backend_name = 'tensorflow'
+
+    @staticmethod
+    def tensor(data, dtype=np.float32, device=None, device_id=None):
+        if isinstance(data, tf.sparse.SparseTensor):
+            return data
+        elif isinstance(data, tuple):
```

Comment:
This `elif` branch is necessary in order to create tensors without having to import tensorflow. Importing tensorflow is what was causing the Travis builds to fail.